etc/travis/suites/application/before_script.sh

```diff
@@ -9,4 +9,4 @@ run_command "bin/console doctrine:migrations:migrate --no-interaction --env=test
 
 print_header "Setting the web assets up" "sylius"
 run_command "bin/console assets:install --env=test_cached --no-debug -vvv" || exit $?
-run_command "npm run gulp" || exit $?
+run_command "npm run yarn run gulp" || exit $?
```

Comment:
Don't know what exactly Travis does, but node, while available in `before_install`, is missing in `install` for example.

Sourcing `nvm.sh` works but can't be a viable solution...
https://github.com/venyii/Sylius/commit/0140c2e37e34fa392e9996676bf00117c0821001
https://travis-ci.org/venyii/Sylius/jobs/207348842